lesson8-UnitTesting/RecentlyUsedList/tests/RecentlyUsedList.Test.cs

```diff
@@ -0,0 +1,91 @@
+using NUnit.Framework;
+using System;
+
+namespace RecentlyUsedList.Tests
+{
+    [TestFixture]
+    public class RecentlyUsedListTest
+    {
+        [Test]
+        public void AddElement()
+        {
+            var list = new RecentlyUsedListSolver();
```

Comment:
Creating of the entity can be moved to SetUp.

Reply:
done